pkg/web/templates/components/select_msg_modal.html

```diff
@@ -0,0 +1,32 @@
+<dialog id="select_msg_modal" class="modal">
+  <div class="modal-box max-w-4xl max-h-screen">
+    <div class="flex justify-between items-center">
+      <h1 class="mx-auto text-2xl font-bold">Select Message Type</h1>
+      <button onclick="select_msg_modal.close()" class="btn btn-sm btn-circle btn-ghost">
```

Comment:
In the acceptance criteria, an outline appears around this button. I'm trying to confirm if this is because of an active style. Whenever I click inside or outside the modal, the outline disappears. I'll have to keep digging to figure out what's causing this because the issue doesn't happen with other modals that have the same styles applied.